docker/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+composer validate
+
+composer install --no-interaction --prefer-dist
+
+php ./vendor/bin/phpinsights -v --no-interaction
+
+php ./vendor/bin/phpstan analyse --ansi --memory-limit 512M
+
+php ./vendor/bin/psalm
+
+php -d error_reporting='E_ALL ^ E_DEPRECATED' -d session.use_cookies=false -d session.cache_limiter=false ./vendor/bin/pest --stop-on-failure --coverage
```

Comment:
I'm curious, I would expect this line to match the line you are using in the CI configuration but I see some differences. Is that accurate?

Reply:
Good catch; the Docker images, as well as the local tests, run PEST/PHPUnit in a slightly different configuration. CircleCI includes extra configuration to store the coverage data for the mutation testing to pick up in a separate task, otherwise the mutation tests would need to run the unit tests a second time to get that data itself when the task runs. It's just caching that data for CircleCI purposes, basically, which isn't necessary on local or Docker testing.